### PR TITLE
Fix/run multiple blocks

### LINKED ISSUE #29: Execute several blocks in a row
Currently the program only executes one block reliably

### PR DESCRIPTION
**Pull request by MaikePaetzel**
The robot is now able to perform multiple walks and turns in a row. The ball behavior still needs to be updated in a similar fashion, though.

### CODE REVIEW

**Comment by schra** [comment]
Very nice :) I like your approach with `isActive`.

I found three problems:

* This example yields infinite rotation (target rotation seems to not be reached?)

  Screenshot from 2021-04-16 22-50-19

* This example yields a JS error:

  Screenshot from 2021-04-16 22-48-43

* Micha implemented that when a player "goes over" a ball, the ball comes with the player (but he didn't implement a smooth transition of the ball - the ball just jumped with the player). This doesn't work anymore.

File: src/helper/useBlockly.js [hunk 1]
```diff
@@ -27,22 +30,74 @@ export function useBlockly() {
...
+                // meisten anderen blöcke, sollte es ausreichen, wenn wir paar ms
+                // warten
+                //console.log("lastBlockType", lastBlockType);
+                const rob = robotList[0];
+                //console.log(rob.isActive);
+                if (rob.isActive) {
+                    return;
+                }
+                workspaceCodeInterpreter.step();
+                workspaceCodeInterpreter.step();
```
Comment (schra) [inline src/helper/useBlockly.js]: I'm unsure whether there is *always* a highlight function after each function in the generated code (e.g. in the case of loops). My approach was to call `.step` only once and divide the interval by two. This way we are sure to never step over non-highlighting block.

Comment (MaikePaetzel) [reply src/helper/useBlockly.js]: Wasn't actually necessary to do that, I took the second call out (was for testing and I forgot to remove it again)

**Comment by MaikePaetzel** [comment]
>     * This example yields infinite rotation (target rotation seems to not be reached?)

Fixed left rotation.

>     * This example yields a JS error:

Was actually a problem with the left rotation as well, so fixed as well.

>     * Micha implemented that when a player "goes over" a ball, the ball comes with the player (but he didn't implement a smooth transition of the ball - the ball just jumped with the player). This doesn't work anymore.

The entire ball interaction doesn't work yet (see my initial message in the PR). Someone else is welcome to take the example of how I implemented the behavior update as part of the Reducer and do the same changes for the ball :)

**Review by schra — CHANGES_REQUESTED** [review]
I get a JS error with this code:

Screenshot from 2021-04-16 23-47-39


```
Uncaught TypeError: rob is undefined
    nextStep useBlockly.js:82
    interpret useBlockly.js:88
    tick useInterval.js:20
useBlockly.js:82
    nextStep useBlockly.js:82
    interpret useBlockly.js:88
    tick useInterval.js:20
```

**Pull request merged**